telegram/utils/helpers.py

```diff
@@ -55,6 +56,79 @@ def get_signal_name(signum: int) -> str:
     return _signames[signum]
 
 
+def is_local_file(obj: Optional[Union[str, Path]], absolute: bool = True) -> bool:
+    """
+    Checks if a given string is a file on local system.
+
+    Args:
+        obj (:obj:`str`): The string to check.
+        absolute (:obj:`bool`): Optional. Whether to allow only absolute paths. Defaults to
+            :obj:`True`.
+    """
+    if obj is None:
+        return False
+
+    path = Path(obj) if isinstance(obj, str) else obj
+    try:
+        if path.exists() and path.is_file():
+            return path.is_absolute() if absolute else True
+        return False
+    except Exception:
+        return False
+
+
+def parse_file_input(
+    file_input: Union[FileInput, 'TelegramObject'],
+    tg_type: Type['TelegramObject'] = None,
+    attach: bool = None,
+    filename: str = None,
+) -> Union[str, 'InputFile', Any]:
+    """
+    Parses input for sending files:
+
+    * For string input, if the input is an absolute path of a local file,
+      adds the ``file://`` prefix. If the input is a relative path of a local file, computes the
+      absolute path and adds the ``file://`` prefix. Returns the input unchanged, otherwise.
+    * :class:`pathlib.Path` objects are treated similarly as strings.
+    * For IO input, returns an :class:`telegram.InputFile`.
+    * If :attr:`tg_type` is specified and the input is of that type, returns the ``file_id``
+      attribute.
+
+    Args:
+        file_input (:obj:`str` | `filelike object` | Telegram media object): The input to parse.
+        tg_type (:obj:`type`, optional): The Telegram media type the input can be. E.g.
+            :class:`telegram.Animation`.
+        attach (:obj:`bool`, optional): Whether this file should be send as one file or is part of
+            a collection of files. Only relevant in case an :class:`telegram.InputFile` is
+            returned.
+        filename (:obj:`str`, optional): The filename. Only relevant in case an
+            :class:`telegram.InputFile` is returned.
+
+    Returns:
+        :obj:`str` | :class:`telegram.InputFile` | :obj:`object`: The parsed input or the untouched
+            :attr:`file_input`, in case it's no valid file input.
+    """
+    # Importing on file-level yields cyclic Import Errors
+    from telegram import InputFile  # pylint: disable=C0415
+
+    if isinstance(file_input, str) and file_input.startswith('file://'):
+        return file_input
+    if isinstance(file_input, (str, Path)):
+        if is_local_file(file_input, absolute=True):
+            out = f'file://{file_input}'
```

Comment:
I don't think we should care if the file is absolute or not.
I think that this will be good for both cases:
```python
if is_local_file(file_input):
    out = f'file://{Path(file_input).absolute()}'
```